ERROR 2025-08-05 17:32:07,825 exception 26640 23676 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
WARNING 2025-08-05 17:32:07,998 log 26640 23676 Bad Request: /api/v1/auth/register/
ERROR 2025-08-05 17:40:59,762 exception 1500 45264 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\http\request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
WARNING 2025-08-05 17:40:59,942 log 1500 45264 Bad Request: /api/v1/auth/register/
INFO 2025-08-06 01:35:31,267 autoreload 19252 11392 Watching for file changes with StatReloader
INFO 2025-08-06 01:45:55,859 basehttp 19252 10492 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
ERROR 2025-08-06 01:45:56,810 log 19252 10492 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 01:45:56,810 basehttp 19252 10492 "POST /api/v1/auth/register/ HTTP/1.1" 500 147
ERROR 2025-08-06 02:47:14,314 log 19252 54444 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 02:47:14,316 basehttp 19252 54444 "POST /api/v1/auth/register/ HTTP/1.1" 500 147
INFO 2025-08-06 02:55:32,125 autoreload 19252 11392 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 02:55:36,028 autoreload 41296 37204 Watching for file changes with StatReloader
INFO 2025-08-06 02:56:47,288 autoreload 41296 37204 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 02:56:50,340 autoreload 19404 26404 Watching for file changes with StatReloader
INFO 2025-08-06 02:58:21,454 autoreload 19404 26404 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 02:58:24,503 autoreload 768 14864 Watching for file changes with StatReloader
INFO 2025-08-06 02:59:21,008 autoreload 768 14864 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 02:59:33,015 autoreload 48108 180 Watching for file changes with StatReloader
ERROR 2025-08-06 03:00:35,306 log 48108 18560 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 03:00:35,309 basehttp 48108 18560 "POST /api/v1/auth/register/ HTTP/1.1" 500 87
INFO 2025-08-06 03:09:18,805 autoreload 48108 180 C:\cursor\smart-account\backend\application\accounts\use_cases.py changed, reloading.
INFO 2025-08-06 03:09:23,662 autoreload 6820 27784 Watching for file changes with StatReloader
INFO 2025-08-06 03:18:07,248 autoreload 40712 34948 Watching for file changes with StatReloader
ERROR 2025-08-06 03:19:32,941 log 40712 36824 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 03:19:32,941 basehttp 40712 36824 "POST /api/v1/auth/register/ HTTP/1.1" 500 120
INFO 2025-08-06 03:23:27,534 autoreload 40712 34948 C:\cursor\smart-account\backend\application\accounts\use_cases.py changed, reloading.
INFO 2025-08-06 03:23:30,658 autoreload 15588 4268 Watching for file changes with StatReloader
INFO 2025-08-06 03:32:55,061 autoreload 18776 24804 Watching for file changes with StatReloader
ERROR 2025-08-06 03:34:38,568 log 18776 29508 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 03:34:38,568 basehttp 18776 29508 "POST /api/v1/auth/register/ HTTP/1.1" 500 97
INFO 2025-08-06 03:38:05,463 autoreload 18776 24804 C:\cursor\smart-account\backend\application\accounts\use_cases.py changed, reloading.
INFO 2025-08-06 03:38:09,625 autoreload 20112 14340 Watching for file changes with StatReloader
INFO 2025-08-06 03:39:13,460 autoreload 16668 54440 Watching for file changes with StatReloader
ERROR 2025-08-06 03:41:19,835 log 16668 15148 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 03:41:19,835 basehttp 16668 15148 "POST /api/v1/auth/register/ HTTP/1.1" 500 98
INFO 2025-08-06 03:45:24,005 autoreload 16668 54440 C:\cursor\smart-account\backend\domain\accounts\services.py changed, reloading.
INFO 2025-08-06 03:45:28,547 autoreload 18304 47172 Watching for file changes with StatReloader
INFO 2025-08-06 03:47:32,691 autoreload 46544 25808 Watching for file changes with StatReloader
INFO 2025-08-06 03:49:27,878 basehttp 46544 8176 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
ERROR 2025-08-06 03:49:28,995 log 46544 8176 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 03:49:29,004 basehttp 46544 8176 "POST /api/v1/auth/register/ HTTP/1.1" 500 334
INFO 2025-08-06 03:54:34,487 autoreload 46544 25808 C:\cursor\smart-account\backend\domain\accounts\services.py changed, reloading.
INFO 2025-08-06 03:54:37,705 autoreload 33496 29864 Watching for file changes with StatReloader
INFO 2025-08-06 10:56:11,133 autoreload 7020 39024 Watching for file changes with StatReloader
INFO 2025-08-06 10:57:24,855 autoreload 54068 7944 Watching for file changes with StatReloader
INFO 2025-08-06 10:58:55,513 autoreload 9488 23152 Watching for file changes with StatReloader
INFO 2025-08-06 11:00:08,455 basehttp 9488 15772 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
ERROR 2025-08-06 11:00:09,821 log 9488 15772 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 11:00:09,822 basehttp 9488 15772 "POST /api/v1/auth/register/ HTTP/1.1" 500 122
INFO 2025-08-06 11:05:01,630 autoreload 9488 23152 C:\cursor\smart-account\backend\domain\accounts\entities.py changed, reloading.
INFO 2025-08-06 11:05:04,270 autoreload 53540 6872 Watching for file changes with StatReloader
INFO 2025-08-06 11:10:33,446 autoreload 53540 6872 C:\cursor\smart-account\backend\domain\accounts\entities.py changed, reloading.
INFO 2025-08-06 11:10:35,896 autoreload 31604 36384 Watching for file changes with StatReloader
INFO 2025-08-06 11:12:05,803 autoreload 20668 54776 Watching for file changes with StatReloader
ERROR 2025-08-06 11:13:05,355 log 20668 41108 Internal Server Error: /api/v1/auth/register/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\me\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\me\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\me\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\utils\encoders.py", line 67, in default
    return super().default(obj)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\me\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Email is not JSON serializable
ERROR 2025-08-06 11:13:05,438 basehttp 20668 41108 "POST /api/v1/auth/register/ HTTP/1.1" 500 130409
INFO 2025-08-06 11:16:31,578 autoreload 20668 54776 C:\cursor\smart-account\backend\application\accounts\use_cases.py changed, reloading.
INFO 2025-08-06 11:16:33,961 autoreload 23208 25632 Watching for file changes with StatReloader
INFO 2025-08-06 11:49:13,645 basehttp 23208 15904 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
ERROR 2025-08-06 11:49:13,736 log 23208 15904 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 11:49:13,737 basehttp 23208 15904 "POST /api/v1/auth/login/ HTTP/1.1" 500 133
INFO 2025-08-06 11:59:24,340 autoreload 23208 25632 C:\cursor\smart-account\backend\infrastructure\email\services.py changed, reloading.
INFO 2025-08-06 11:59:28,657 autoreload 32648 42796 Watching for file changes with StatReloader
INFO 2025-08-06 12:04:12,717 autoreload 32648 42796 C:\cursor\smart-account\backend\smart_accounts\settings\development.py changed, reloading.
INFO 2025-08-06 12:04:16,138 autoreload 9380 52112 Watching for file changes with StatReloader
INFO 2025-08-06 12:14:42,459 autoreload 9380 52112 C:\cursor\smart-account\backend\smart_accounts\settings\development.py changed, reloading.
INFO 2025-08-06 12:14:45,123 autoreload 27408 34944 Watching for file changes with StatReloader
INFO 2025-08-06 12:27:44,783 autoreload 18532 28220 Watching for file changes with StatReloader
ERROR 2025-08-06 12:27:51,351 log 18532 15852 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 12:27:51,352 basehttp 18532 15852 "POST /api/v1/auth/login/ HTTP/1.1" 500 133
INFO 2025-08-06 12:34:16,192 autoreload 18532 28220 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 12:34:19,530 autoreload 9068 15128 Watching for file changes with StatReloader
INFO 2025-08-06 12:34:58,832 autoreload 9068 15128 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 12:35:01,593 autoreload 49944 14932 Watching for file changes with StatReloader
INFO 2025-08-06 12:36:56,419 autoreload 34720 30804 Watching for file changes with StatReloader
ERROR 2025-08-06 12:38:01,107 log 34720 49132 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 12:38:01,108 basehttp 34720 49132 "POST /api/v1/auth/login/ HTTP/1.1" 500 93
INFO 2025-08-06 12:40:44,088 autoreload 34720 30804 C:\cursor\smart-account\backend\domain\accounts\services.py changed, reloading.
INFO 2025-08-06 12:40:46,781 autoreload 40060 36988 Watching for file changes with StatReloader
INFO 2025-08-06 12:43:40,145 autoreload 46888 37660 Watching for file changes with StatReloader
ERROR 2025-08-06 12:44:21,753 log 46888 15276 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 12:44:21,754 basehttp 46888 15276 "POST /api/v1/auth/login/ HTTP/1.1" 500 104
INFO 2025-08-06 12:46:44,339 autoreload 46888 37660 C:\cursor\smart-account\backend\domain\accounts\entities.py changed, reloading.
INFO 2025-08-06 12:46:47,075 autoreload 1228 40576 Watching for file changes with StatReloader
INFO 2025-08-06 12:47:14,142 autoreload 1228 40576 C:\cursor\smart-account\backend\domain\accounts\entities.py changed, reloading.
INFO 2025-08-06 12:47:16,643 autoreload 11700 44524 Watching for file changes with StatReloader
INFO 2025-08-06 12:48:09,142 autoreload 30152 48204 Watching for file changes with StatReloader
ERROR 2025-08-06 12:48:36,051 log 30152 41308 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 12:48:36,052 basehttp 30152 41308 "POST /api/v1/auth/login/ HTTP/1.1" 500 89
INFO 2025-08-06 12:51:07,600 autoreload 30152 48204 C:\cursor\smart-account\backend\application\accounts\use_cases.py changed, reloading.
INFO 2025-08-06 12:51:10,419 autoreload 16908 10952 Watching for file changes with StatReloader
INFO 2025-08-06 13:00:35,963 autoreload 44548 7224 Watching for file changes with StatReloader
ERROR 2025-08-06 13:00:56,298 log 44548 416 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 13:00:56,299 basehttp 44548 416 "POST /api/v1/auth/login/ HTTP/1.1" 500 61
INFO 2025-08-06 13:03:52,730 autoreload 44548 7224 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 13:03:55,408 autoreload 52484 41180 Watching for file changes with StatReloader
INFO 2025-08-06 13:04:29,739 autoreload 52484 41180 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 13:04:32,197 autoreload 31412 55120 Watching for file changes with StatReloader
INFO 2025-08-06 13:12:16,284 autoreload 31472 54328 Watching for file changes with StatReloader
INFO 2025-08-06 13:12:59,391 basehttp 31472 42724 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
INFO 2025-08-06 13:14:14,775 basehttp 31472 42724 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
INFO 2025-08-06 13:23:09,521 basehttp 31472 23368 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
INFO 2025-08-06 13:23:10,457 basehttp 31472 23368 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 13:43:15,977 basehttp 31472 8064 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
INFO 2025-08-06 13:47:18,172 basehttp 31472 8064 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 13:54:34,478 autoreload 31472 54328 C:\cursor\smart-account\backend\interfaces\api\serializers.py changed, reloading.
INFO 2025-08-06 13:54:37,298 autoreload 47768 43516 Watching for file changes with StatReloader
INFO 2025-08-06 13:55:45,315 autoreload 47768 43516 C:\cursor\smart-account\backend\interfaces\api\serializers.py changed, reloading.
INFO 2025-08-06 13:55:47,926 autoreload 6720 51232 Watching for file changes with StatReloader
INFO 2025-08-06 14:03:02,793 autoreload 8796 9472 Watching for file changes with StatReloader
INFO 2025-08-06 14:46:45,200 autoreload 46756 324 Watching for file changes with StatReloader
ERROR 2025-08-06 14:47:42,352 log 46756 40732 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 14:47:42,353 basehttp 46756 40732 "POST /api/v1/auth/register/ HTTP/1.1" 500 89
ERROR 2025-08-06 14:48:46,599 log 46756 40732 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 14:48:46,600 basehttp 46756 40732 "POST /api/v1/auth/register/ HTTP/1.1" 500 89
ERROR 2025-08-06 14:52:26,395 log 46756 40732 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 14:52:26,395 basehttp 46756 40732 "POST /api/v1/auth/register/ HTTP/1.1" 500 89
INFO 2025-08-06 15:02:48,326 autoreload 46756 324 C:\cursor\smart-account\backend\domain\common\entities.py changed, reloading.
INFO 2025-08-06 15:02:51,081 autoreload 51620 35980 Watching for file changes with StatReloader
INFO 2025-08-06 15:04:03,006 autoreload 51620 35980 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 15:04:05,688 autoreload 38040 36308 Watching for file changes with StatReloader
INFO 2025-08-06 15:05:15,420 autoreload 38040 36308 C:\cursor\smart-account\backend\interfaces\api\serializers.py changed, reloading.
INFO 2025-08-06 15:05:18,756 autoreload 37644 11900 Watching for file changes with StatReloader
INFO 2025-08-06 15:06:27,977 autoreload 37644 11900 C:\cursor\smart-account\backend\interfaces\api\urls.py changed, reloading.
INFO 2025-08-06 15:06:30,724 autoreload 3976 47432 Watching for file changes with StatReloader
INFO 2025-08-06 15:07:37,709 autoreload 3976 47432 C:\cursor\smart-account\backend\interfaces\api\urls.py changed, reloading.
INFO 2025-08-06 15:07:40,360 autoreload 2600 14532 Watching for file changes with StatReloader
INFO 2025-08-06 15:09:57,456 autoreload 2600 14532 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 15:10:00,184 autoreload 52944 7040 Watching for file changes with StatReloader
INFO 2025-08-06 16:01:59,951 basehttp 52944 41096 "POST /api/v1/auth/register/ HTTP/1.1" 201 203
INFO 2025-08-06 16:02:03,075 basehttp 52944 9180 "POST /api/v1/auth/register/ HTTP/1.1" 201 201
ERROR 2025-08-06 16:02:06,294 log 52944 25232 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 16:02:06,295 basehttp 52944 25232 "POST /api/v1/auth/login/ HTTP/1.1" 500 106
ERROR 2025-08-06 16:02:09,378 log 52944 10940 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 16:02:09,379 basehttp 52944 10940 "POST /api/v1/auth/login/ HTTP/1.1" 500 106
ERROR 2025-08-06 16:02:11,646 log 52944 45556 Internal Server Error: /api/v1/auth/password-reset/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 208, in post
    serializer = PasswordResetRequestSerializer(data=request.data)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'PasswordResetRequestSerializer' is not defined
ERROR 2025-08-06 16:02:11,683 basehttp 52944 45556 "POST /api/v1/auth/password-reset/ HTTP/1.1" 500 119363
INFO 2025-08-06 16:07:22,536 autoreload 52944 7040 C:\cursor\smart-account\backend\smart_accounts\settings\development.py changed, reloading.
INFO 2025-08-06 16:07:28,229 autoreload 38720 30556 Watching for file changes with StatReloader
INFO 2025-08-06 16:12:22,102 autoreload 30172 38584 Watching for file changes with StatReloader
INFO 2025-08-06 16:23:26,450 basehttp 30172 11788 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
ERROR 2025-08-06 16:23:28,283 log 30172 11788 Internal Server Error: /api/v1/auth/register/
ERROR 2025-08-06 16:23:28,284 basehttp 30172 11788 "POST /api/v1/auth/register/ HTTP/1.1" 500 107
INFO 2025-08-06 16:30:03,369 autoreload 45068 3100 Watching for file changes with StatReloader
INFO 2025-08-06 16:31:39,606 autoreload 21604 40856 Watching for file changes with StatReloader
INFO 2025-08-06 16:32:48,506 basehttp 21604 24204 "POST /api/v1/auth/register/ HTTP/1.1" 201 203
INFO 2025-08-06 16:32:51,750 basehttp 21604 39932 "POST /api/v1/auth/register/ HTTP/1.1" 201 201
INFO 2025-08-06 16:32:55,625 basehttp 21604 42192 "POST /api/v1/auth/login/ HTTP/1.1" 200 262
INFO 2025-08-06 16:32:58,818 basehttp 21604 4520 "POST /api/v1/auth/login/ HTTP/1.1" 200 270
ERROR 2025-08-06 16:33:00,975 log 21604 14216 Internal Server Error: /api/v1/auth/password-reset/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 208, in post
    serializer = PasswordResetRequestSerializer(data=request.data)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'PasswordResetRequestSerializer' is not defined
ERROR 2025-08-06 16:33:01,011 basehttp 21604 14216 "POST /api/v1/auth/password-reset/ HTTP/1.1" 500 119362
INFO 2025-08-06 16:35:32,257 basehttp 21604 46764 "POST /api/v1/auth/register/ HTTP/1.1" 201 203
INFO 2025-08-06 16:35:35,440 basehttp 21604 51068 "POST /api/v1/auth/register/ HTTP/1.1" 201 201
INFO 2025-08-06 16:35:38,457 basehttp 21604 7608 "POST /api/v1/auth/login/ HTTP/1.1" 200 262
INFO 2025-08-06 16:35:41,472 basehttp 21604 49360 "POST /api/v1/auth/login/ HTTP/1.1" 200 270
INFO 2025-08-06 16:35:44,507 basehttp 21604 3032 "POST /api/v1/auth/register/ HTTP/1.1" 201 198
ERROR 2025-08-06 16:35:46,610 log 21604 30768 Internal Server Error: /api/v1/auth/password-reset/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 208, in post
    serializer = PasswordResetRequestSerializer(data=request.data)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'PasswordResetRequestSerializer' is not defined
ERROR 2025-08-06 16:35:46,641 basehttp 21604 30768 "POST /api/v1/auth/password-reset/ HTTP/1.1" 500 119361
INFO 2025-08-06 16:36:42,379 autoreload 21604 40856 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 16:36:45,768 autoreload 20236 36404 Watching for file changes with StatReloader
INFO 2025-08-06 16:36:55,736 basehttp 20236 13960 "POST /api/v1/auth/register/ HTTP/1.1" 201 203
INFO 2025-08-06 16:36:58,947 basehttp 20236 41552 "POST /api/v1/auth/register/ HTTP/1.1" 201 201
INFO 2025-08-06 16:37:02,394 basehttp 20236 12696 "POST /api/v1/auth/login/ HTTP/1.1" 200 262
INFO 2025-08-06 16:37:05,361 basehttp 20236 44568 "POST /api/v1/auth/login/ HTTP/1.1" 200 270
INFO 2025-08-06 16:37:08,445 basehttp 20236 14256 "POST /api/v1/auth/register/ HTTP/1.1" 201 198
ERROR 2025-08-06 16:37:10,617 log 20236 51352 Internal Server Error: /api/v1/auth/password-reset/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 208, in post
    serializer = PasswordResetRequestSerializer(data=request.data)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'PasswordResetRequestSerializer' is not defined
ERROR 2025-08-06 16:37:10,657 basehttp 20236 51352 "POST /api/v1/auth/password-reset/ HTTP/1.1" 500 119362
INFO 2025-08-06 16:37:33,040 autoreload 20236 36404 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-06 16:37:35,777 autoreload 43368 32912 Watching for file changes with StatReloader
WARNING 2025-08-06 16:50:23,141 log 43368 11556 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 16:50:23,142 basehttp 43368 11556 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
WARNING 2025-08-06 16:54:01,791 log 43368 31932 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 16:54:01,792 basehttp 43368 31932 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
INFO 2025-08-06 17:01:04,039 basehttp 43368 40664 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 17:02:32,289 basehttp 43368 52968 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
INFO 2025-08-06 18:48:48,586 autoreload 36568 45428 Watching for file changes with StatReloader
INFO 2025-08-06 18:49:30,709 basehttp 36568 29244 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
WARNING 2025-08-06 18:49:31,317 log 36568 29244 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 18:49:31,318 basehttp 36568 29244 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
INFO 2025-08-06 18:55:17,102 basehttp 36568 45328 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 18:55:38,445 basehttp 36568 11912 "POST /api/v1/auth/register/ HTTP/1.1" 201 200
WARNING 2025-08-06 18:55:40,580 log 36568 15100 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 18:55:40,581 basehttp 36568 15100 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
WARNING 2025-08-06 18:55:42,717 log 36568 26804 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 18:55:42,718 basehttp 36568 26804 "POST /api/v1/auth/register/ HTTP/1.1" 400 113
WARNING 2025-08-06 18:55:44,863 log 36568 54692 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 18:55:44,864 basehttp 36568 54692 "POST /api/v1/auth/register/ HTTP/1.1" 400 113
ERROR 2025-08-06 19:28:23,769 log 36568 19340 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 19:28:23,770 basehttp 36568 19340 "POST /api/v1/auth/login/ HTTP/1.1" 500 71
INFO 2025-08-06 19:40:41,221 basehttp 36568 31932 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 19:40:44,246 basehttp 36568 12152 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
ERROR 2025-08-06 19:40:46,316 log 36568 51492 Internal Server Error: /api/v1/auth/verify-email/
ERROR 2025-08-06 19:40:46,317 basehttp 36568 51492 "POST /api/v1/auth/verify-email/ HTTP/1.1" 500 96
INFO 2025-08-06 19:41:51,351 autoreload 47956 45760 Watching for file changes with StatReloader
WARNING 2025-08-06 19:42:26,355 log 47956 34152 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 19:42:26,356 basehttp 47956 34152 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
INFO 2025-08-06 19:56:15,494 basehttp 47956 29080 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 19:56:18,297 basehttp 47956 21208 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
INFO 2025-08-06 19:56:44,433 basehttp 47956 21924 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
INFO 2025-08-06 19:56:45,219 basehttp 47956 21924 "POST /api/v1/auth/login/ HTTP/1.1" 200 265
WARNING 2025-08-06 19:58:24,697 log 47956 21924 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 19:58:24,698 basehttp 47956 21924 "POST /api/v1/auth/register/ HTTP/1.1" 400 137
INFO 2025-08-06 21:03:53,513 autoreload 3160 21452 Watching for file changes with StatReloader
INFO 2025-08-06 21:04:37,778 basehttp 3160 37544 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 0
WARNING 2025-08-06 21:04:38,430 log 3160 37544 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 21:04:38,431 basehttp 3160 37544 "POST /api/v1/auth/register/ HTTP/1.1" 400 137
WARNING 2025-08-06 21:07:58,586 log 3160 37544 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 21:07:58,592 basehttp 3160 37544 "POST /api/v1/auth/register/ HTTP/1.1" 400 137
INFO 2025-08-06 21:10:11,796 basehttp 3160 25064 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
WARNING 2025-08-06 21:17:54,050 log 3160 13768 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 21:17:54,051 basehttp 3160 13768 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
INFO 2025-08-06 21:17:57,327 basehttp 3160 51884 "POST /api/v1/auth/register/ HTTP/1.1" 201 198
INFO 2025-08-06 21:23:49,026 basehttp 3160 51248 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
WARNING 2025-08-06 21:54:07,467 log 3160 45104 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 21:54:07,468 basehttp 3160 45104 "POST /api/v1/auth/register/ HTTP/1.1" 400 117
WARNING 2025-08-06 21:58:43,272 log 3160 54044 Method Not Allowed: /api/v1/auth/register/
WARNING 2025-08-06 21:58:43,274 basehttp 3160 54044 "GET /api/v1/auth/register/ HTTP/1.1" 405 19500
INFO 2025-08-06 21:58:43,318 basehttp 3160 15132 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2025-08-06 21:58:43,321 basehttp 3160 31036 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2025-08-06 21:58:43,326 basehttp 3160 54044 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
INFO 2025-08-06 21:58:43,342 basehttp 3160 15132 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
INFO 2025-08-06 21:58:43,346 basehttp 3160 31036 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 12621
INFO 2025-08-06 21:58:43,382 basehttp 3160 31036 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1719
INFO 2025-08-06 21:58:43,390 basehttp 3160 15132 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2025-08-06 21:58:43,459 basehttp 3160 15132 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2025-08-06 21:58:43,466 basehttp 3160 31036 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
INFO 2025-08-06 21:58:43,494 basehttp 3160 54044 "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 200 89476
INFO 2025-08-06 21:58:43,505 basehttp 3160 15132 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
INFO 2025-08-06 21:58:43,520 basehttp 3160 54044 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2025-08-06 21:58:43,523 basehttp 3160 31036 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 13404
INFO 2025-08-06 21:58:43,557 basehttp 3160 31036 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
INFO 2025-08-06 21:58:43,593 basehttp 3160 54044 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4527
WARNING 2025-08-06 21:58:43,711 log 3160 54044 Not Found: /favicon.ico
WARNING 2025-08-06 21:58:43,712 basehttp 3160 54044 "GET /favicon.ico HTTP/1.1" 404 12451
WARNING 2025-08-06 21:59:30,973 log 3160 54044 Bad Request: /api/v1/auth/register/
WARNING 2025-08-06 21:59:30,975 basehttp 3160 54044 "POST /api/v1/auth/register/ HTTP/1.1" 400 19645
INFO 2025-08-06 21:59:31,194 basehttp 3160 54044 "GET /__debug__/history_sidebar/?store_id=6dfb6a8bb45e4944bee9a2f946a49b3b HTTP/1.1" 200 9107
INFO 2025-08-06 21:59:50,179 basehttp 3160 14944 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
INFO 2025-08-06 21:59:50,912 basehttp 3160 14944 "POST /api/v1/auth/login/ HTTP/1.1" 200 273
INFO 2025-08-06 22:00:01,559 basehttp 3160 14944 "POST /api/v1/auth/login/ HTTP/1.1" 200 273
INFO 2025-08-06 22:00:42,927 basehttp 3160 14944 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 22:00:47,984 basehttp 3160 14944 "POST /api/v1/auth/login/ HTTP/1.1" 200 261
ERROR 2025-08-06 22:12:51,346 log 3160 20128 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 22:12:51,381 basehttp 3160 20128 "POST /api/v1/auth/login/ HTTP/1.1" 500 71
INFO 2025-08-06 22:13:36,520 basehttp 3160 20128 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 22:13:45,479 basehttp 3160 20128 "POST /api/v1/auth/login/ HTTP/1.1" 200 273
INFO 2025-08-06 22:14:25,237 basehttp 3160 20128 "POST /api/v1/auth/register/ HTTP/1.1" 201 202
INFO 2025-08-06 22:14:28,479 basehttp 3160 20128 "POST /api/v1/auth/login/ HTTP/1.1" 200 261
INFO 2025-08-06 23:12:53,915 basehttp 3160 1988 "POST /api/v1/auth/login/ HTTP/1.1" 200 261
INFO 2025-08-06 23:13:07,852 basehttp 3160 1988 "POST /api/v1/auth/login/ HTTP/1.1" 200 273
INFO 2025-08-06 23:46:18,046 basehttp 3160 11520 "POST /api/v1/auth/login/ HTTP/1.1" 200 273
INFO 2025-08-06 23:46:32,937 basehttp 3160 11520 "POST /api/v1/auth/login/ HTTP/1.1" 200 261
ERROR 2025-08-06 23:47:50,490 log 3160 22984 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 23:47:50,491 basehttp 3160 22984 "POST /api/v1/auth/login/ HTTP/1.1" 500 80
ERROR 2025-08-06 23:53:08,073 log 3160 49488 Internal Server Error: /api/v1/auth/login/
ERROR 2025-08-06 23:53:08,073 basehttp 3160 49488 "POST /api/v1/auth/login/ HTTP/1.1" 500 80
INFO 2025-08-06 23:55:54,175 basehttp 3160 47676 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
INFO 2025-08-06 23:56:12,163 basehttp 3160 19412 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
INFO 2025-08-06 23:56:23,510 basehttp 3160 48872 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
INFO 2025-08-06 23:56:57,381 basehttp 3160 46360 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
INFO 2025-08-06 23:59:14,002 basehttp 3160 51880 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
WARNING 2025-08-06 23:59:16,991 log 3160 45320 Unauthorized: /api/v1/receipts/upload/
WARNING 2025-08-06 23:59:16,992 basehttp 3160 45320 "POST /api/v1/receipts/upload/ HTTP/1.1" 401 183
INFO 2025-08-07 00:04:41,876 basehttp 3160 34536 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
WARNING 2025-08-07 00:05:27,043 log 3160 19172 Unauthorized: /api/v1/receipts/upload/
WARNING 2025-08-07 00:05:27,044 basehttp 3160 19172 "POST /api/v1/receipts/upload/ HTTP/1.1" 401 183
INFO 2025-08-07 00:11:50,261 basehttp 3160 17944 "POST /api/v1/auth/login/ HTTP/1.1" 200 268
WARNING 2025-08-07 00:12:24,433 log 3160 48072 Unauthorized: /api/v1/receipts/upload/
WARNING 2025-08-07 00:12:24,435 basehttp 3160 48072 "POST /api/v1/receipts/upload/ HTTP/1.1" 401 183
INFO 2025-08-07 00:14:00,532 autoreload 3160 21452 C:\cursor\smart-account\backend\domain\accounts\services.py changed, reloading.
INFO 2025-08-07 00:14:03,233 autoreload 32684 16492 Watching for file changes with StatReloader
INFO 2025-08-07 00:16:40,147 autoreload 32684 16492 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 00:16:44,790 autoreload 26892 20892 Watching for file changes with StatReloader
INFO 2025-08-07 00:19:17,918 autoreload 26892 20892 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 00:19:20,497 autoreload 18880 31596 Watching for file changes with StatReloader
INFO 2025-08-07 00:22:01,053 autoreload 18880 31596 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 00:22:03,623 autoreload 50420 26544 Watching for file changes with StatReloader
WARNING 2025-08-07 00:26:19,992 log 50420 39760 Not Found: /api/auth/login/
WARNING 2025-08-07 00:26:19,993 basehttp 50420 39760 "POST /api/auth/login/ HTTP/1.1" 404 12469
INFO 2025-08-07 00:29:17,582 basehttp 50420 52168 "POST /api/v1/auth/login/ HTTP/1.1" 200 776
INFO 2025-08-07 00:32:25,258 basehttp 50420 25952 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
INFO 2025-08-07 00:32:27,233 basehttp 50420 25952 "POST /api/v1/auth/login/ HTTP/1.1" 200 770
WARNING 2025-08-07 00:34:55,387 log 50420 30984 Not Found: /api/v1/api/v1/auth/login/
WARNING 2025-08-07 00:34:55,388 basehttp 50420 30984 "POST /api/v1/api/v1/auth/login/ HTTP/1.1" 404 17210
INFO 2025-08-07 00:37:50,771 autoreload 50420 26544 C:\cursor\smart-account\backend\domain\accounts\services.py changed, reloading.
INFO 2025-08-07 00:37:53,343 autoreload 42092 46400 Watching for file changes with StatReloader
INFO 2025-08-07 00:41:37,068 basehttp 42092 45792 "POST /api/v1/auth/login/ HTTP/1.1" 200 777
ERROR 2025-08-07 00:43:08,695 log 42092 43356 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 442, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 00:43:08,741 basehttp 42092 43356 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126144
INFO 2025-08-07 00:43:33,696 basehttp 42092 26492 "POST /api/v1/auth/login/ HTTP/1.1" 200 777
ERROR 2025-08-07 00:45:23,570 log 42092 22044 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 442, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 00:45:23,626 basehttp 42092 22044 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126144
INFO 2025-08-07 00:49:33,759 basehttp 42092 47080 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
INFO 2025-08-07 00:59:07,820 autoreload 4348 54832 Watching for file changes with StatReloader
INFO 2025-08-07 00:59:57,697 basehttp 4348 52480 "POST /api/v1/auth/login/ HTTP/1.1" 200 770
INFO 2025-08-07 01:01:39,882 basehttp 4348 53612 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 19808
INFO 2025-08-07 01:01:40,103 basehttp 4348 53612 "GET /__debug__/history_sidebar/?store_id=1002977ab6da4a02a053d8fd67640686 HTTP/1.1" 200 9112
INFO 2025-08-07 01:01:42,125 basehttp 4348 53612 "OPTIONS /api/v1/auth/register/ HTTP/1.1" 200 19807
INFO 2025-08-07 01:01:42,344 basehttp 4348 53612 "GET /__debug__/history_sidebar/?store_id=0576cb176097496c9c4ffd34623ea7fd HTTP/1.1" 200 9111
WARNING 2025-08-07 01:01:45,301 log 4348 53612 Method Not Allowed: /api/v1/auth/register/
WARNING 2025-08-07 01:01:45,302 basehttp 4348 53612 "GET /api/v1/auth/register/ HTTP/1.1" 405 19499
WARNING 2025-08-07 01:01:45,434 log 4348 53612 Not Found: /favicon.ico
WARNING 2025-08-07 01:01:45,435 basehttp 4348 53612 "GET /favicon.ico HTTP/1.1" 404 12451
WARNING 2025-08-07 01:43:43,657 log 4348 20676 Not Found: /api/v1/health/
WARNING 2025-08-07 01:43:43,658 basehttp 4348 20676 "GET /api/v1/health/ HTTP/1.1" 404 17165
INFO 2025-08-07 01:45:26,019 autoreload 4348 54832 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-07 01:45:29,968 autoreload 44524 180 Watching for file changes with StatReloader
INFO 2025-08-07 01:48:16,902 autoreload 44524 180 C:\cursor\smart-account\backend\interfaces\api\urls.py changed, reloading.
INFO 2025-08-07 01:48:20,105 autoreload 42632 1960 Watching for file changes with StatReloader
INFO 2025-08-07 01:49:41,706 basehttp 42632 25620 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 01:49:47,048 basehttp 42632 27516 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 01:49:49,129 log 42632 27308 Unauthorized: /api/v1/receipts/upload/
WARNING 2025-08-07 01:49:49,130 basehttp 42632 27308 "OPTIONS /api/v1/receipts/upload/ HTTP/1.1" 401 58
ERROR 2025-08-07 01:49:51,427 log 42632 46188 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 457, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 01:49:51,458 basehttp 42632 46188 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126369
INFO 2025-08-07 01:59:39,103 basehttp 42632 51300 "GET / HTTP/1.1" 200 48
INFO 2025-08-07 01:59:41,171 basehttp 42632 44840 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 01:59:44,246 basehttp 42632 35200 "POST /api/v1/auth/register/ HTTP/1.1" 201 200
INFO 2025-08-07 01:59:47,806 basehttp 42632 42536 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
ERROR 2025-08-07 01:59:49,950 log 42632 20944 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 457, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 01:59:49,975 basehttp 42632 20944 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126365
INFO 2025-08-07 02:06:23,340 basehttp 42632 22272 "POST /api/v1/auth/login/ HTTP/1.1" 200 780
ERROR 2025-08-07 02:06:25,565 log 42632 30368 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 457, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 02:06:25,593 basehttp 42632 30368 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126369
INFO 2025-08-07 02:06:48,869 autoreload 42632 1960 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-07 02:06:51,781 autoreload 48668 53556 Watching for file changes with StatReloader
INFO 2025-08-07 02:25:10,056 basehttp 48668 29124 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
INFO 2025-08-07 02:34:44,326 autoreload 50160 3820 Watching for file changes with StatReloader
INFO 2025-08-07 02:36:16,155 basehttp 50160 48948 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
INFO 2025-08-07 02:36:17,595 basehttp 50160 48948 "POST /api/v1/auth/login/ HTTP/1.1" 200 770
INFO 2025-08-07 02:36:37,603 basehttp 50160 48948 "POST /api/v1/auth/login/ HTTP/1.1" 200 770
INFO 2025-08-07 02:46:25,854 basehttp 50160 13736 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 02:50:20,166 basehttp 50160 24668 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 02:50:33,598 basehttp 50160 19840 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
ERROR 2025-08-07 02:50:37,567 log 50160 11628 Internal Server Error: /api/v1/receipts/upload/
Traceback (most recent call last):
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\backend\interfaces\api\views.py", line 457, in post
    serializer = ReceiptUploadSerializer(data=request.data, files=request.FILES)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cursor\smart-account\venv\Lib\site-packages\rest_framework\serializers.py", line 116, in __init__
    super().__init__(**kwargs)
TypeError: Field.__init__() got an unexpected keyword argument 'files'
ERROR 2025-08-07 02:50:38,229 basehttp 50160 11628 "POST /api/v1/receipts/upload/ HTTP/1.1" 500 126367
INFO 2025-08-07 02:51:08,514 basehttp 50160 20768 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
INFO 2025-08-07 02:51:34,335 basehttp 50160 22952 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
INFO 2025-08-07 02:52:49,080 basehttp 50160 20768 "GET /api/v1/health/ HTTP/1.1" 200 98
WARNING 2025-08-07 02:52:49,157 log 50160 20768 Unauthorized: /api/v1/receipts/upload/
WARNING 2025-08-07 02:52:49,158 basehttp 50160 20768 "POST /api/v1/receipts/upload/ HTTP/1.1" 401 58
INFO 2025-08-07 03:02:23,371 autoreload 50160 3820 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-07 03:02:26,544 autoreload 35428 52496 Watching for file changes with StatReloader
INFO 2025-08-07 03:09:50,297 basehttp 35428 25424 "POST /api/v1/auth/login/ HTTP/1.1" 200 778
INFO 2025-08-07 03:10:05,938 autoreload 35428 52496 C:\cursor\smart-account\backend\interfaces\api\views.py changed, reloading.
INFO 2025-08-07 03:10:09,436 autoreload 14540 47148 Watching for file changes with StatReloader
INFO 2025-08-07 03:14:17,637 autoreload 16116 40752 Watching for file changes with StatReloader
INFO 2025-08-07 03:20:48,701 basehttp 16116 22324 "POST /api/v1/auth/login/ HTTP/1.1" 200 772
WARNING 2025-08-07 03:21:16,729 log 16116 27500 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:21:16,729 basehttp 16116 27500 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:21:31,753 log 16116 19864 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:21:31,753 basehttp 16116 19864 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:21:47,188 log 16116 31104 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:21:47,189 basehttp 16116 31104 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:22:06,575 log 16116 49872 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:22:06,576 basehttp 16116 49872 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:22:23,961 log 16116 33672 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:22:23,961 basehttp 16116 33672 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:22:37,739 log 16116 28976 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:22:37,740 basehttp 16116 28976 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:22:52,383 log 16116 30808 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:22:52,384 basehttp 16116 30808 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:23:08,612 log 16116 6456 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:23:08,613 basehttp 16116 6456 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:23:23,299 log 16116 18520 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:23:23,300 basehttp 16116 18520 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:23:40,773 log 16116 40820 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:23:40,775 basehttp 16116 40820 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:24:01,311 log 16116 984 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:24:01,312 basehttp 16116 984 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:24:27,170 log 16116 14944 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:24:27,171 basehttp 16116 14944 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:24:45,864 log 16116 21100 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:24:45,865 basehttp 16116 21100 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:25:04,385 log 16116 41276 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:25:04,386 basehttp 16116 41276 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:25:28,361 log 16116 35772 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:25:28,362 basehttp 16116 35772 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
WARNING 2025-08-07 03:25:46,679 log 16116 38048 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 03:25:46,680 basehttp 16116 38048 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 15:26:04,671 autoreload 25076 31288 Watching for file changes with StatReloader
INFO 2025-08-07 15:27:26,711 basehttp 25076 28116 "OPTIONS /api/v1/auth/login/ HTTP/1.1" 200 0
INFO 2025-08-07 15:27:29,860 basehttp 25076 28116 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
INFO 2025-08-07 15:27:49,672 basehttp 25076 28116 "OPTIONS /api/v1/health/ HTTP/1.1" 200 0
INFO 2025-08-07 15:27:49,778 basehttp 25076 28116 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 15:27:49,783 basehttp 25076 28116 "OPTIONS /api/v1/receipts/upload/ HTTP/1.1" 200 0
WARNING 2025-08-07 15:28:18,126 log 25076 28116 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 15:28:18,127 basehttp 25076 28116 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 15:33:58,292 autoreload 25076 31288 C:\cursor\smart-account\venv\Lib\site-packages\pytz\lazy.py changed, reloading.
INFO 2025-08-07 15:34:03,614 autoreload 54060 6556 Watching for file changes with StatReloader
INFO 2025-08-07 15:37:17,760 autoreload 54060 6556 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 15:37:24,419 autoreload 51420 25448 Watching for file changes with StatReloader
INFO 2025-08-07 15:40:19,028 autoreload 51420 25448 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 15:40:22,306 autoreload 28480 47584 Watching for file changes with StatReloader
INFO 2025-08-07 15:43:46,782 autoreload 28480 47584 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 15:43:53,677 autoreload 52496 35020 Watching for file changes with StatReloader
INFO 2025-08-07 16:12:19,036 autoreload 52496 35020 C:\cursor\smart-account\backend\domain\receipts\entities.py changed, reloading.
INFO 2025-08-07 16:12:21,958 autoreload 35620 34464 Watching for file changes with StatReloader
INFO 2025-08-07 16:15:19,769 autoreload 35620 34464 C:\cursor\smart-account\backend\domain\receipts\entities.py changed, reloading.
INFO 2025-08-07 16:15:22,641 autoreload 40484 23428 Watching for file changes with StatReloader
INFO 2025-08-07 16:18:24,988 autoreload 40484 23428 C:\cursor\smart-account\backend\domain\receipts\entities.py changed, reloading.
INFO 2025-08-07 16:18:27,834 autoreload 32512 13028 Watching for file changes with StatReloader
INFO 2025-08-07 16:34:54,760 autoreload 32512 13028 C:\cursor\smart-account\backend\domain\receipts\entities.py changed, reloading.
INFO 2025-08-07 16:34:57,249 autoreload 15776 20032 Watching for file changes with StatReloader
INFO 2025-08-07 18:03:02,904 autoreload 32092 2240 Watching for file changes with StatReloader
INFO 2025-08-07 18:11:00,411 autoreload 32092 2240 C:\cursor\smart-account\backend\application\receipts\use_cases.py changed, reloading.
INFO 2025-08-07 18:11:05,901 autoreload 10024 20004 Watching for file changes with StatReloader
INFO 2025-08-07 18:13:54,206 autoreload 10024 20004 C:\cursor\smart-account\backend\application\receipts\use_cases.py changed, reloading.
INFO 2025-08-07 18:13:57,551 autoreload 34952 1740 Watching for file changes with StatReloader
INFO 2025-08-07 19:17:36,652 autoreload 34952 1740 C:\cursor\smart-account\backend\application\receipts\use_cases.py changed, reloading.
INFO 2025-08-07 19:17:39,207 autoreload 49204 36672 Watching for file changes with StatReloader
INFO 2025-08-07 19:24:12,415 basehttp 49204 47408 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:24:45,072 basehttp 49204 30240 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:25:09,155 log 49204 41232 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 19:25:09,156 basehttp 49204 41232 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 19:34:35,851 basehttp 49204 49828 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:35:08,250 basehttp 49204 29432 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:35:32,650 log 49204 21796 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 19:35:32,652 basehttp 49204 21796 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 19:37:33,062 autoreload 52312 12312 Watching for file changes with StatReloader
INFO 2025-08-07 19:38:25,568 basehttp 52312 30364 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:38:56,749 basehttp 52312 40520 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:39:25,284 log 52312 14464 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 19:39:25,286 basehttp 52312 14464 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 19:39:53,167 basehttp 52312 42632 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:40:22,373 basehttp 52312 13456 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:40:39,037 log 52312 10332 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 19:40:39,038 basehttp 52312 10332 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 19:45:28,569 autoreload 49856 15936 Watching for file changes with StatReloader
INFO 2025-08-07 19:46:12,504 basehttp 49856 36756 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:47:06,254 basehttp 49856 16920 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:47:57,200 log 49856 26980 Bad Request: /api/v1/receipts/upload/
INFO 2025-08-07 19:47:57,201 basehttp 49856 26980 - Broken pipe from ('127.0.0.1', 55333)
INFO 2025-08-07 19:49:57,195 basehttp 49856 14852 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 19:50:43,712 basehttp 49856 26388 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 19:51:06,801 log 49856 26340 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 19:51:06,802 basehttp 49856 26340 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 19:54:35,749 basehttp 49856 16860 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 20:05:09,718 basehttp 49856 21796 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 20:05:34,746 log 49856 51148 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 20:05:34,748 basehttp 49856 51148 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 20:06:07,158 basehttp 49856 52668 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 20:15:35,074 basehttp 49856 16688 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 20:15:53,358 log 49856 4288 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 20:15:53,359 basehttp 49856 4288 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 20:31:52,653 autoreload 31344 35664 Watching for file changes with StatReloader
INFO 2025-08-07 20:32:39,039 basehttp 31344 41972 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 20:43:58,433 basehttp 31344 25852 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 20:44:23,708 log 31344 22000 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 20:44:23,709 basehttp 31344 22000 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 20:58:28,399 autoreload 31344 35664 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 20:58:31,355 autoreload 3908 32984 Watching for file changes with StatReloader
INFO 2025-08-07 20:58:58,113 autoreload 3908 32984 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 20:59:00,678 autoreload 34840 50352 Watching for file changes with StatReloader
INFO 2025-08-07 20:59:48,510 autoreload 49796 11668 Watching for file changes with StatReloader
INFO 2025-08-07 21:00:28,879 basehttp 49796 39140 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 21:07:04,130 basehttp 49796 47196 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 21:07:47,147 log 49796 26388 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 21:07:47,161 basehttp 49796 26388 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 22:42:39,317 autoreload 49796 11668 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 22:42:42,555 autoreload 18420 27564 Watching for file changes with StatReloader
INFO 2025-08-07 22:46:10,158 autoreload 18420 27564 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 22:46:12,856 autoreload 13268 20128 Watching for file changes with StatReloader
INFO 2025-08-07 22:46:59,427 autoreload 13268 20128 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 22:47:01,879 autoreload 23372 43120 Watching for file changes with StatReloader
INFO 2025-08-07 22:48:24,371 autoreload 31060 19664 Watching for file changes with StatReloader
INFO 2025-08-07 23:01:28,970 autoreload 31060 19664 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 23:01:32,641 autoreload 3600 19384 Watching for file changes with StatReloader
INFO 2025-08-07 23:02:13,106 autoreload 27972 19184 Watching for file changes with StatReloader
INFO 2025-08-07 23:14:28,898 autoreload 27972 19184 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-07 23:14:31,505 autoreload 53348 44684 Watching for file changes with StatReloader
INFO 2025-08-07 23:14:53,713 autoreload 32832 16692 Watching for file changes with StatReloader
INFO 2025-08-07 23:15:24,863 basehttp 32832 41048 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 23:21:19,129 basehttp 32832 20756 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 23:21:37,366 log 32832 16440 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 23:21:37,367 basehttp 32832 16440 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-07 23:29:00,253 autoreload 17752 26168 Watching for file changes with StatReloader
INFO 2025-08-07 23:30:06,013 basehttp 17752 34904 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 23:42:35,963 autoreload 19592 33376 Watching for file changes with StatReloader
INFO 2025-08-07 23:42:52,345 basehttp 19592 29276 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-07 23:55:08,006 basehttp 19592 13772 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-07 23:55:20,380 log 19592 17040 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-07 23:55:20,381 basehttp 19592 17040 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 00:24:00,787 autoreload 19592 33376 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-08 00:24:04,195 autoreload 25988 12104 Watching for file changes with StatReloader
INFO 2025-08-08 00:24:41,332 autoreload 46540 38316 Watching for file changes with StatReloader
INFO 2025-08-08 00:25:23,660 basehttp 46540 44004 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 00:39:53,188 autoreload 13244 15532 Watching for file changes with StatReloader
INFO 2025-08-08 00:50:51,557 autoreload 13244 15532 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-08 00:51:23,224 autoreload 45156 7876 Watching for file changes with StatReloader
INFO 2025-08-08 01:07:36,423 basehttp 45156 53332 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 01:10:15,848 basehttp 45156 41016 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 01:20:35,203 autoreload 45156 7876 C:\cursor\smart-account\backend\infrastructure\ocr\enhanced_paddle_ocr.py changed, reloading.
INFO 2025-08-08 01:21:01,607 autoreload 21012 29272 Watching for file changes with StatReloader
INFO 2025-08-08 01:21:16,041 autoreload 21012 29272 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-08 01:21:19,133 autoreload 35796 35612 Watching for file changes with StatReloader
INFO 2025-08-08 01:22:04,453 autoreload 39076 51028 Watching for file changes with StatReloader
INFO 2025-08-08 02:48:59,875 autoreload 39076 51028 C:\cursor\smart-account\backend\infrastructure\ocr\services.py changed, reloading.
INFO 2025-08-08 02:49:02,992 autoreload 44284 15924 Watching for file changes with StatReloader
INFO 2025-08-08 02:49:18,013 autoreload 26080 41232 Watching for file changes with StatReloader
INFO 2025-08-08 03:04:07,113 autoreload 4272 15676 Watching for file changes with StatReloader
INFO 2025-08-08 03:04:54,192 basehttp 4272 16456 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 03:05:35,203 basehttp 4272 46076 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 03:05:57,914 log 4272 7220 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 03:05:57,915 basehttp 4272 7220 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 03:28:26,432 autoreload 45896 23504 Watching for file changes with StatReloader
INFO 2025-08-08 03:28:59,527 basehttp 45896 24860 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 03:30:14,544 basehttp 45896 42396 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 03:31:00,422 log 45896 22476 Bad Request: /api/v1/receipts/upload/
INFO 2025-08-08 03:31:00,426 basehttp 45896 22476 - Broken pipe from ('127.0.0.1', 53171)
INFO 2025-08-08 03:42:46,895 autoreload 31712 42224 Watching for file changes with StatReloader
INFO 2025-08-08 03:44:16,747 basehttp 31712 25484 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 03:44:48,287 basehttp 31712 9556 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 03:45:14,273 log 31712 27464 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 03:45:14,274 basehttp 31712 27464 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 03:50:15,511 autoreload 40860 38916 Watching for file changes with StatReloader
INFO 2025-08-08 03:50:54,542 basehttp 40860 35432 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 03:51:37,186 basehttp 40860 32144 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 03:52:01,323 log 40860 26748 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 03:52:01,324 basehttp 40860 26748 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 04:42:07,867 autoreload 17332 35916 Watching for file changes with StatReloader
INFO 2025-08-08 04:42:58,550 basehttp 17332 10540 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 04:43:40,052 basehttp 17332 27984 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 04:44:08,285 log 17332 8976 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 04:44:08,286 basehttp 17332 8976 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 04:52:28,011 autoreload 30440 32224 Watching for file changes with StatReloader
INFO 2025-08-08 04:53:10,963 basehttp 30440 23048 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 04:53:53,899 basehttp 30440 50400 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 04:54:18,879 log 30440 15520 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 04:54:18,880 basehttp 30440 15520 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 05:04:50,354 autoreload 9192 50752 Watching for file changes with StatReloader
INFO 2025-08-08 05:05:47,265 basehttp 9192 37884 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 05:06:30,559 basehttp 9192 40336 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 05:06:54,405 log 9192 53900 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 05:06:54,406 basehttp 9192 53900 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 05:26:07,246 autoreload 41364 43668 Watching for file changes with StatReloader
INFO 2025-08-08 05:26:37,276 basehttp 41364 26360 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 05:27:11,063 basehttp 41364 41412 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 05:27:35,195 log 41364 22124 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 05:27:35,196 basehttp 41364 22124 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 05:31:57,841 basehttp 41364 55256 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 05:32:41,905 basehttp 41364 39140 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 05:33:12,623 log 41364 34888 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 05:33:12,626 basehttp 41364 34888 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 05:40:21,754 autoreload 3360 29032 Watching for file changes with StatReloader
INFO 2025-08-08 05:40:39,707 basehttp 3360 15924 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 05:43:55,216 basehttp 3360 10940 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 05:44:26,157 log 3360 6000 Bad Request: /api/v1/receipts/upload/
WARNING 2025-08-08 05:44:26,158 basehttp 3360 6000 "POST /api/v1/receipts/upload/ HTTP/1.1" 400 46
INFO 2025-08-08 05:47:05,494 autoreload 12492 7160 Watching for file changes with StatReloader
INFO 2025-08-08 05:48:48,813 basehttp 12492 33860 "GET /api/v1/health/ HTTP/1.1" 200 98
INFO 2025-08-08 05:52:02,378 basehttp 12492 28264 "POST /api/v1/auth/login/ HTTP/1.1" 200 782
WARNING 2025-08-08 05:52:36,182 log 12492 17904 Bad Request: /api/v1/receipts/upload/
INFO 2025-08-08 05:52:36,184 basehttp 12492 17904 - Broken pipe from ('127.0.0.1', 63171)
INFO 2025-08-08 12:54:35,839 autoreload 22384 44172 Watching for file changes with StatReloader
